content/en/docs/Contribution Guidelines/releasing.md

```diff
@@ -173,3 +173,18 @@ Update the downloads page on parquet.apache.org. Instructions for updating the s
     Java artifacts are available from Maven Central.
 
     Thanks to everyone for contributing!
+
+#### 6\. Update parquet-format with feature enablement guidance
+
+The recommendations for other feature enablement is generally tied to releases of
+parquet-java (details are in the parquet-format repo).  As releases are made the
+specification should be updated to indicate the recommended dates for when a new
+feature may be enabled.
+
+
+### Release Cadence
+
+Provided enough volunteers are available the Parquet community aims to have
+releases on a quarterly basis (Targets months are January, April, July and
+October). If a new major version is necessary it will be targetted for the
```

Comment:
I'm fine to release a major version in every October.

However, as I have mentioned in https://github.com/apache/parquet-format/pull/258#discussion_r1660557601, a lot of deprecations are marked to be removed in 2.0.0 like https://github.com/apache/parquet-java/blob/e6d7728267691a1ff433358b6c60d0cae38b5e36/parquet-common/src/main/java/org/apache/parquet/Closeables.java#L29.

Perhaps this October we should just release 1.15.0 to keep the API compatibility. We have enough time to prepare for 2.0.0 next year.

cc @gszadovszky @Fokko @julienledem

Reply:
TBH, I am not a big fun of doing a major release only to remove deprecated parts. But let's discuss the 2.0.0 content and release schedule on the dev list.

Reply:
I do think it is time for a new major release to remove a lot of legacy (for example the removal of OriginalType https://github.com/apache/parquet-java/pull/1194). However, we don't want to be too aggressive, so this is a balance we have to find. I do think adding a 1.15.0 release might smooth out things, for example, by raising the lower bound of Hadoop step by step (>2.9 in 1.15 in https://github.com/apache/parquet-java/pull/2944, and then we can move to >3.3 at some point in 2.0?)

Reply:
We could communicate the dates to the public by setting dates on the milestones: https://github.com/apache/parquet-java/milestones

Reply:
Rebased.